presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/common/PicTextField.kt

```diff
@@ -0,0 +1,82 @@
+package com.mashup.gabbangzip.sharedalbum.presentation.ui.common
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.Cultured
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.Gray80
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.PicTypography
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.SharedAlbumTheme
+import com.mashup.gabbangzip.sharedalbum.presentation.theme.SilverSand
+
+@Composable
+fun PicTextField(
+    modifier: Modifier = Modifier,
+    value: String,
+    onValueChange: (String) -> Unit,
+    hint: String,
+    maxLength: Int = 0,
+) {
+    BasicTextField(
+        modifier = modifier
+            .background(
+                color = Cultured,
+                shape = RoundedCornerShape(10.dp),
+            )
+            .padding(horizontal = 20.dp, vertical = 18.dp),
+        value = value,
+        onValueChange = {
+            if (maxLength == 0 || it.length <= maxLength) {
```

Comment:
```suggestion
            if (maxLength == 0 || it.length < maxLength) {
```
<= maxLength 이면 (maxLength + 1) 까지 입력될 것 같은데..? 확인이 필요할거같아!

Reply:
maxLength가 10이라고 했을 때 10글자까지는 바꿔주고 11글자부터 안 바꿔주는거라 maxLength 포함해야함다 ~!

Reply:
오케이 확인~~